components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -65,41 +65,76 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
+                                className='advisor-icon'
+                                src={ackIcon}
+                            />
+                            <FormattedMarkdownMessage
+                                id={t('announcement_bar.error.number_active_users_warn_metric_status_ack.text')}
+                                defaultMessage={'Your trial has started! Go to the [System Console](/admin_console/environment/web_server) to check out the new features.'}
+                            />
+                        </React.Fragment>
+                    );
+                } else {
+                    message = (
+                        <React.Fragment>
+                            <img
+                                className='advisor-icon'
+                                src={ackIcon}
+                            />
+                            <FormattedMarkdownMessage
+                                id={t('announcement_bar.warn_metric_status_ack.text')}
+                                defaultMessage={'Thank you for contacting Mattermost. We will follow up with you soon.'}
+                            />
+                        </React.Fragment>
+                    );
+                }
+
                 type = AnnouncementBarTypes.ADVISOR_ACK;
                 showModal = false;
                 dismissFunc = this.dismissNumberOfActiveUsersWarnMetricAck;
                 isDismissed = this.props.dismissedNumberOfActiveUsersWarnMetricStatusAck;
                 canCloseBar = true;
             } else {
-                message = (
-                    <React.Fragment>
-                        <img
-                            className='advisor-icon'
-                            src={alertIcon}
-                        />
-                        <FormattedMarkdownMessage
-                            id={t('announcement_bar.error.number_active_users_warn_metric_status.text')}
-                            defaultMessage={'You now have over {limit} users. We strongly recommend using advanced features for large-scale servers.'}
-                            values={{
-                                limit: warnMetricStatus.limit,
-                            }}
-                        />
-                    </React.Fragment>
-                );
+                if (warnMetricStatus.id === WarnMetricTypes.SYSTEM_WARN_METRIC_NUMBER_OF_ACTIVE_USERS_500) {
+                    message = (
+                        <React.Fragment>
+                            <img
+                                className='advisor-icon'
+                                src={alertIcon}
+                            />
+                            <FormattedMarkdownMessage
+                                id={t('announcement_bar.error.number_active_users_warn_metric_status.text')}
+                                defaultMessage={'You now have over {limit} users. We strongly recommend using advanced features for large-scale servers.'}
```

Comment:
```suggestion
                                defaultMessage='You now have over {limit} users. We strongly recommend using advanced features for large-scale servers.'
```

Reply:
Users are more likely to take action if you tell them exactly what you want them to do and why. This is so vague we're making them interpret it for themselves and it's not clear what we're asking of them.

Reply:
I'll defer to @esadur and @jasonblais on this, they review the texts.

Reply:
These are the banners, and there's only so much space for text. We go into more detail in the modals.